tests/testthat/test-sce_to_anndata.R

```diff
@@ -3,10 +3,13 @@ sce <- sim_sce(n_cells = 100, n_genes = 200, n_empty = 0)
 colData(sce) <- DataFrame("test_column" = sample(0:10, 100, rep = TRUE))
 rowData(sce) <- DataFrame("test_row" = sample(0:10, 200, rep = TRUE))
 
+# define anndata output
+anndata_file <- "test_anndata.h5"
+# quiet messages that will come with zellkonverter
+suppressPackageStartupMessages(library(zellkonverter))
```

Comment:
For convention, we should probably move the package loading to the top of the file.